docs/components.md

```diff
@@ -57,6 +57,24 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used. The component's model events do not bubble up.
+
+To respond to events on a component's model, a parent or ancestor application/component can listen to the event on the model but will have to scope the listening action to the component. Custom events on the component's model must be exposed to its ancestor via the `exposedEvents` object.
+
+```javascript
+var ClockModel = BaseModel.extend({
+  postInitialize: function() {
+    this.listenTo(this, 'change:time_component:est_time', this.updateTime)
```

Comment:
we should show and explain why this is not such a good idea:

`this.listenTo(this.get('time_component'), 'change:est_time', this.updateTime);`